### ISSUE TITLE
My pod's CocoaPod metrics are wrong.

### ISSUE THREAD

**Issue by WilliamZang** [CocoaPods/cocoapods.org#386]
- [x] I've read and understood the *CONTRIBUTING guidelines and have done my best effort to follow.

# Report
## What did you do?
Invoked https://metrics.cocoapods.org/api/v1/pods/EasyReact.json

# What did you expected to happen?
Expected to see updated stats for today, star count of Github should be 1200+ but only 6.

# What happened instead?
Star count of Github is 6.

# CocoaPods Environment
N/A

# Project that demonstrates the issue
https://metrics.cocoapods.org/api/v1/pods/EasyReact.json

**Comment by RastislavMirek** [comment]
Yes, star metrcis are not updated. I should have 19 starts by now but zero is shown on the website. Link: https://cocoapods.org/pods/FlexColorPicker
